src/pymovements/events/ivt.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+from typing import Optional
+import numpy as np
+
+from pymovements.transforms import vnorm
+from pymovements.events import Fixation
+
+
+def ivt(
+        x: list[list[float]] | np.ndarray,
+        v: Optional[list[list[float]] | np.ndarray],
+        t: float
+) -> list[Fixation]:
+    """
+    Identification of fixations based on velocity-threshold
```

Comment:
Please explain better how exactly ivt works, especially how `t` is used

Reply:
Also add a citation to original paper.

For this to look best, do the following:
- Add a bib entry to https://github.com/aeye-lab/pymovements/blob/main/docs/source/bibliography.bib
- Copy the complete bibtex from https://dl.acm.org/doi/10.1145/355017.355028
- Use `Salvucci2000` as the bib-key.
- Create a citation in the docstring using :cite:t:`Salvucci2000`.

Here's a reference for using bibtex in sphinx:
https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html